torch/lib/c10d/test/TCPStoreTest.cpp

```diff
@@ -10,19 +11,27 @@
 #include <c10d/TCPStore.hpp>
 
 constexpr int64_t kShortStoreTimeoutMillis = 100;
+constexpr int64_t kStoreCallbackTimeoutMillis = 5000;
+constexpr int defaultTimeout = 20;
 
-// Different ports for different tests.
-void testHelper(const std::string& prefix = "") {
-  const auto numThreads = 16;
-  const auto numWorkers = numThreads + 1;
-
-  auto serverTCPStore = c10::make_intrusive<c10d::TCPStore>(
+c10::intrusive_ptr<c10d::TCPStore> _createServer(
+    int numWorkers = 1,
+    int timeout = defaultTimeout) {
+  return c10::make_intrusive<c10d::TCPStore>(
       "127.0.0.1",
       0,
       numWorkers,
       true,
-      std::chrono::seconds(30),
+      std::chrono::seconds(timeout),
       /* wait */ false);
+}
+
+// Different ports for different tests.
+void testHelper(const std::string& prefix = "") {
+  const auto numThreads = 16;
+  const auto numWorkers = numThreads + 1;
+
+  auto serverTCPStore = _createServer(numWorkers);
```

Comment:
Would I be correct if I assume the only change in this test function is extracting the server creation logic into a dedicated `_createServer` function? Other changes are mostly clang-format? (except push_back -> emplace_back)

Reply:
Yep, that's correct those are the only changes